draft-ietf-quic-transport.md

```diff
@@ -994,6 +994,22 @@ packets sent from only one local address.  An endpoint that migrates away from a
 local address SHOULD retire all connection IDs used on that address once it no
 longer plans to use that address.
 
+An endpoint can request that its peer retire connection IDs by sending a
+NEW_CONNECTION_ID frame with an increased Retire Prior To field.  Upon receipt,
+the peer SHOULD retire the corresponding connection IDs and send the
+corresponding RETIRE_CONNECTION_ID frame in a timely manner.  Failing to do so
+can cause packets to be delayed or lost and harm connection performance as the
+original endpoint might not route those connection IDs optimally after some
+delay.
+
+The sender of the Retire Prior To field MUST keep track of the connection IDs it
```

Comment:
I still think this MUST is unenforceable and a bit odd.  If my load balancer restarts in 5 seconds and the peer hasn't retired the old CIDs by then, they're still getting retired.

Reply:
Change it to a `SHOULD`?

Reply:
It's currently a MUST to persist all the CIDs with the same SRT.  The way to be compliant would be to advance RPT enough to expire all the CIDs, even if there are some in the middle that could otherwise be kept.